echo_server.py

```diff
@@ -3,6 +3,58 @@
 import socket
 
 
+def response_ok():
+    """Return byte string 200 ok response."""
+    return u"HTTP/1.1 200 OK\r\nContent-Type: text/plain\r\nContent-length: 18\r\n\r\neverything is okay".encode('utf-8')
```

Comment:
This line is too long.  But the function is right.  Just fix the length of the line.